win32/static_filters.src.cpp

```diff
@@ -0,0 +1,216 @@
+/*File generated during static filter build
+  Automatically generated file
+*/
```

Comment:
Why is this file checked in?  It should be generated automatically.

Reply:
See above:

- perl script mk-static-filter.pl to create static_filters.src.cpp still has a yet unresolved problem therefore win32\static_filters.src.cpp was added

Reply:
It would be better to resolve the issues, this file is already outdated with the new markdown filter.  Please file a separate issue for this with a patch to fix the generated file and I can likely fix the script if required.